web/impact/impact/tests/test_algolia_api_key_view.py

```diff
@@ -122,6 +126,30 @@ def test_user_with_staff_role_grant_sees_all_mentors(self):
                 response_data = json.loads(response.content)
                 self.assertEqual(response_data["filters"], [])
 
+    def test_staff_user_does_not_have_is_active_filter(self):
+        user = self.basic_user()
+        program = _create_batch_program_and_named_group(
+                        ACTIVE_PROGRAM_STATUS, 1)
+        self._create_user_with_role_grant(program[0], UserRole.STAFF, user)
+        with self.settings(
+                ALGOLIA_APPLICATION_ID='test',
+                ALGOLIA_API_KEY='test'):
+            with self.login(email=user.email):
+                response = self.client.get(self._person_directory_url())
+                response_data = json.loads(response.content)
+                self.assertFalse(IS_ACTIVE_FILTER in response_data["filters"])
+
+    def test_finalist_user_filter_includes_is_active_filter(self):
+        program = ProgramFactory(program_status=ACTIVE_PROGRAM_STATUS)
+        user = self._create_user_with_role_grant(program, UserRole.FINALIST)
+        with self.settings(
+                ALGOLIA_APPLICATION_ID='test',
+                ALGOLIA_API_KEY='test'):
+            with self.login(email=user.email):
+                response = self.client.get(self._person_directory_url())
+                response_data = json.loads(response.content)
+                self.assertTrue(IS_ACTIVE_FILTER in response_data["filters"])
```

Comment:
@mkiterian for Python3's case we might want to use `self.assertIn` or `self.assertNotIn`